main/test/ca/uwaterloo/flix/language/phase/TestTyper.scala

```diff
@@ -1468,39 +1468,6 @@ class TestTyper extends AnyFunSuite with TestUtils {
     expectError[TypeError.MissingTraitConstraint](result)
   }
 
-  test("TypeError.AmbiguousMethod.01") {
-    val input =
-      """
-        |import java.lang.StringBuilder
-        |
-        |def main(): Unit \ IO =
-        |    import java_new java.lang.StringBuilder(String): StringBuilder \ IO as newSB;
-        |    let a = testInvokeMethod2_01(newSB(""));
-        |    println(a.toString())
-        |
-        |def testInvokeMethod2_01(sb: StringBuilder): StringBuilder \ IO =
-        |    sb.append(null)
-        |""".stripMargin
-    val result = compile(input, Options.Default)
-    expectError[TypeError.AmbiguousMethod](result)
-  }
-
-  test("TypeError.AmbiguousMethod.02") {
-    val input =
-      """
-        |import java.io.PrintStream
-        |
-        |def main(): Unit \ IO =
-        |    import java_new java.io.PrintStream(String): PrintStream \ IO as newPS;
-        |    testInvokeMethod2_01(newPS(""))
-        |
-        |def testInvokeMethod2_01(ps: PrintStream): Unit \ IO =
-        |    ps.println(null)
-        |""".stripMargin
-    val result = compile(input, Options.Default)
-    expectError[TypeError.AmbiguousMethod](result)
-  }
-
```

Comment:
I removed these two tests because they look obsolete -- Why are they using java_new???

Reply:
But it would be good to add new tests.. I just don't have the time now.